src/httpPipResolver.js

```diff
@@ -3,19 +3,13 @@
 var util = require('util');
 var http = require('http');
 var request = require('request');
-var peliasConfig = require( 'pelias-config' ).generate();
 var _ = require('lodash');
 
+let maxConcurrentReqs;
```

Comment:
why is this module scope?

I only see one reference to it at the bottom

There is also another variable of another scope with the same name:

```
this.maxConcurrentReqs
```

is that intended?